src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -943,7 +953,7 @@ public void visitSources(SCMSourceObserver observer) throws IOException, Interru
       }
 
       GitHubSCMNavigatorContext gitHubSCMNavigatorContext =
-          new GitHubSCMNavigatorContext().withTraits(traits);
+          new GitHubSCMNavigatorContext().withTraits(getTraits());
```

Comment:
https://github.com/jenkinsci/github-branch-source-plugin/pull/344#discussion_r555225295

Reply:
nvm, looked more carefully at the code

Reply:
So can the PR be merged? 😄